Ovation.FasterQC.Net.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
```

Comment:
i'll bet this was a pain... that's why i don't do it

Reply:
yes, yes it was 😂